File: leetcode/0200-0299/283. Move Zeroes/283.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        n = len(nums)
        if n == 1:
            return
        l = i = 0
        while l < n:
            if nums[l]:
                nums[i], nums[l] = nums[l], nums[i]
                i += 1

            l += 1

File: leetcode/0200-0299/283. Move Zeroes/283.cs
public class Solution
{
    public void MoveZeroes(int[] nums)
    {
        int l = 0, i = 0, n = nums.Length;
        while (l < n)
        {
            if (nums[l] != 0)
            {
                int temp = nums[i];
                nums[i] = nums[l];
                nums[l] = temp;
                i++;
            }
            l++;
        }
    }
}